### DISCUSSION TITLE
Won't Launch or anything...Need Pre-reqs?

### DISCUSSION THREAD

**Discussion by areonic** [UnchartedBull/OctoDash#1730] [General]
Hello, I installed a fresh copy of the latest octoprint on 2 pi's, both with diff screens. Tried on 3.5 and 7". Both will not work, when I try to manually launch it, I get something segmentation or huge gap of xserver stuff. Do I need to install the desktop UI or something before installing octodash?

Thank you

**Comment by TRIBAL2003** [comment]
https://unchartedbull.github.io/OctoDash/index.html
